package.json

```diff
@@ -41,8 +43,11 @@
     "node-simple-logger": "0.0.1",
     "properties": "^1.2.1",
     "react": "^15.3.1",
+    "react-addons-update": "^15.3.0",
+    "react-controllables": "^0.6.0",
     "react-dom": "^15.3.1",
     "react-facebook-login": "^3.1.0",
+    "react-pure-render": "^1.0.2",
```

Comment:
Stop using this. React mixin is depreacted: https://facebook.github.io/react/blog/2016/07/13/mixins-considered-harmful.html